src/schemas/21-526EZ/schema.js

```diff
@@ -371,6 +393,34 @@ let schema = {
         }
       }
     },
+    privateRecordReleases: {
+      type: 'array',
+      items: {
+        type: 'object',
+        required: ['treatmentCenterName'],
+        properties: {
+          treatmentCenterName: {
+            type: 'string',
+            maxLength: 100,
+            pattern: "([a-zA-Z0-9\-'.#]([a-zA-Z0-9\-'.# ])?)+$"
+          },
+          treatmentDateRange: {
+            $ref: '#/definitions/dateRange'
+          },
+          treatmentCenterAddress: {
+            $ref: '#/definitions/privateTreatmentCenterAddress'
+          },
+          privateMedicalRecordsReleaseAccepted: {
+            type: 'boolean'
+          },
+          'view:privateMedicalRecordsReleasePermissionRestricted': {
```

Comment:
'view:...' type objects never gets sent to the backend, so we do not need to include it here, we could however, include its contents, namely `privateMedicalRecordsReleasePermissionRestricted`, which should be a boolean.

Reply:
👍 this was a holdover from the transferring the form properties straight into the schema. Since this needs to get submitted as part of the separate ancillary form submission I'll make it not a `view` field.

Reply:
Oh does this refer to the warning that gets displayed when `privateMedicalRecordsReleaseAccepted` is selected? We can simply remove it if so.

Reply:
The naming here is slightly confusing, perhaps `privateMedicalRecordsReleaseAccepted` should be `privateMedicalRecordsReleaseRestricted`

Reply:
FYI this only controls the text in the conditional alert dialog when a user chooses to restrict their private records request. The boolean that stores whether or not this option is selected is `privateMedicalRecordsReleaseAccepted`

Reply:
Ah missed your other comments :)
Agree that the naming is confusing. I'll try to give these properties names closer to their intent.